src/stratis_cli/_actions/_top.py

```diff
@@ -75,8 +78,8 @@ def list_pools(_):
         tables = [
            [
               mopool.Name(),
-              mopool.TotalPhysicalSize(),
-              mopool.TotalPhysicalUsed()
+              str(Range(mopool.TotalPhysicalSize(), SECTOR_SIZE)),
```

Comment:
Note that you're accepting the default configuration here. The defaults were chosen by me as the most useful for humans representation I could come up with. But I'm not everybody's kind of human.

Reply:
heh. Well it looked ok to me!